lib/screens/v2/candidate_generator/elevator/closures.ex

```diff
@@ -197,7 +261,7 @@ defmodule Screens.V2.CandidateGenerator.Elevator.Closures do
             station_closures,
             fn %Closure{id: id, name: name} -> %WidgetClosure{id: id, name: name} end
           ),
-        summary: active_summary(station_closures, closures)
+        summary: active_summary(station_closures, active_closures)
```

Comment:
The value here was previously "relevant closures", but now is _all_ active closures. It should stay `relevant_closures` so the existing logic of this function remains the same.

Reply:
Actually the value passed here was pre-filtered closures (when `closures` first arrived to the function it was only active closures, and that filtering was done earlier in the pipeline). Using `relevant_closures` here caused some test failures because I guess we weren't able to access the full summary given on the closure. (L409 in `closures_test.exs`)

Reply:
Oh, my bad, this does make sense: When displaying accessibility summaries, we need to know if any of a given elevator's backup elevators are also closed, and that's independent of whether or not the backup elevator was considered "relevant" for the purposes of building the overall closure list. I misread this. Thanks!